app/templating/StringHelper.scala

```diff
@@ -47,8 +47,23 @@ trait StringHelper { self: NumberHelper =>
     }
   }
 
+  // Matches a lichess username with a '@' prefix (I hope.) Example: @ornicar
+  private val atUsernameRegex = "(?<=^|(?<=[^a-zA-Z0-9-_\\.]))@([A-Za-z]+[A-Za-z0-9]+)".r
```

Comment:
can't `\b` be used at the beginning of this regex? instead of `(?<=^|(?<=[^a-zA-Z0-9-_\\.]))`